.eslintrc

```diff
@@ -18,26 +18,31 @@
     "platform_require": true
   },
   "plugins": [
+    "json",
```

Comment:
Added [**eslint-plugin-json**](http://npm.im/eslint-plugin-json) for JSON linting in ESLint.